project_custom_info/demo/default.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Tecnativa - Carlos Roca
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<odoo>
+    <record id="prop_time" model="custom.info.property">
+        <field name="default_value">3</field>
```

Comment:
Why not including this on the CSV?

Reply:
I'm based on base_custom_info In these module, the default value was added like here... I don't know the reason but I thought that's important.

Reply:
Please try the other way to see. I don't find right now the reason for this.